AmplifyPlugins/Auth/AWSCognitoAuthPluginTests/ClientBehaviorAPITests/AWSCognitoAuthClientBehaviorTests.swift

```diff
@@ -8,22 +8,41 @@
 import XCTest
 @testable import Amplify
 @testable import AWSCognitoAuthPlugin
+import Combine
 
-class AWSCognitoAuthClientBehaviorTests: XCTestCase {
+// swiftlint:disable file_length type_body_length
+final class AWSCognitoAuthClientBehaviorTests: XCTestCase {
 
     var plugin: AWSCognitoAuthPlugin!
+    var authenticationProvider: MockAuthenticationProviderBehavior!
+    var authorizationProvider: MockAuthorizationProviderBehavior!
+    var userService: MockAuthUserServiceBehavior!
+    var deviceService: MockAuthDeviceServiceBehavior!
+    var hubEventHandler: MockAuthHubEventBehavior!
 
-    override func setUp() {
+    override func setUpWithError() throws {
+        authenticationProvider = MockAuthenticationProviderBehavior()
+        authorizationProvider = MockAuthorizationProviderBehavior()
+        userService = MockAuthUserServiceBehavior()
+        deviceService = MockAuthDeviceServiceBehavior()
+        hubEventHandler = MockAuthHubEventBehavior()
         plugin = AWSCognitoAuthPlugin()
-        plugin.configure(authenticationProvider: MockAuthenticationProviderBehavior(),
-                         authorizationProvider: MockAuthorizationProviderBehavior(),
-                         userService: MockAuthUserServiceBehavior(),
-                         deviceService: MockAuthDeviceServiceBehavior(),
-                         hubEventHandler: MockAuthHubEventBehavior())
+        plugin.configure(authenticationProvider: authenticationProvider,
+                         authorizationProvider: authorizationProvider,
+                         userService: userService,
+                         deviceService: deviceService,
+                         hubEventHandler: hubEventHandler)
+        try Amplify.configure(AmplifyConfiguration())
```

Comment:
The operations were failing because they were attempting to post Hub messages after `Amplify.reset()` and before any other `Amplify.configure()` call.